Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -720,6 +792,17 @@ - (UIFont *)dateHeaderTimeFont
     return [UIFont systemFontOfSize:12.0f];
 }
 
+- (UILabel *)createQuotedTextView
```

Comment:
nit: `resetQuotedTextView` since we're not alloc'ing anything in this method?

Maybe also `OWSAssert(self.quotedTextView)`

Reply:
Convert this to in fact create the `quotedTextLabel`.